genet/outputs_handler/sanitiser.py

```diff
@@ -18,6 +20,9 @@ def sanitise_geodataframe(gdf):
             gdf[col] = gdf[col].apply(lambda x: ','.join(x))
         elif gdf[col].apply(lambda x: isinstance(x, dict)).any():
             gdf[col] = gdf[col].apply(lambda x: str(x))
+    for col in gdf.select_dtypes(include=number).columns.tolist():
```

Comment:
Does changing the type to a string have any impact on the data in its serialised form? I had a look at generated GeoJSON in Matesto for test town, and I see:

```json
    {
      "type": "Feature",
      "properties": {
        "index": "7",
        "x": 500,
        "y": 3500,
        "id": "7",
        "lon": -7.553841327916719,
        "lat": 49.79847109699483,
        "s2_id": 5206348332324383000
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          -7.553841327916719,
          49.79847109699483
        ]
      }
    }
```

Will this mean `"s2_id": 5206348332324383000` becomes `"s2_id": "5206348332324383000"`? If so, are there any implications for processes that read these GeoJSON files?

Reply:
They only become strings if they're too large, so the test town indices should remain as they are. Nothing apart from kepler or GIS software would be reading these geojson files, I haven't tested it but I think kepler will be ok with those strings.